api/src/main/java/org/openmrs/parameter/OrderSearchCriteria.java

```diff
@@ -50,8 +50,8 @@
 	 * Instead of calling this constructor directly, it is recommended to use {@link OrderSearchCriteriaBuilder}.
 	 * @param patient the patient the order is for
 	 * @param careSetting the care setting to match on
-	 * @param concepts the concepts defining the order must be in this collection
-	 * @param orderTypes the order types to match on must be in this collection
+	 * @param concepts the concepts defining the order must be in this collection if specified, matches on all concepts otherwise
```

Comment:
**Otherwise** seems to have been left hanging here

Reply:
@wluyima this is gramatically correct, unless you mean adding a period at the end?

Reply:
I feel like otherwise is redundant or gives the impression there is another behavior but this is the default, _**if specified, matches on all concepts**_ sounds enough to me, am not a native English speaker so I might be wrong.

Reply:
how about this: **if specified, the concepts defining the order must be in this collection, otherwise matches on all concepts** ?

Reply:
or simply: ** if specified, the concepts defining the order must be in this collection**

Reply:
the "otherwise" is key, because it is "otherwise"... the first phrase seems to be the one that is confusing to me, how abotu:
"the concepts to match on; if not specified, matches on all concepts"
"the order types to match on; if not specified, matches all order types"